config/common.yaml

```diff
@@ -208,18 +212,6 @@ console:
         tls:
           insecureEdgeTerminationPolicy: Redirect
           termination: passthrough
-  persistentVolumeClaims:
-    - metadata:
-        name: "{{.ApplicationName}}-rhpamcentr-claim"
```

Comment:
doesn't the production environment require this claim? other envs?

Reply:
makes sense, was just making sure production was still properly getting this PVC

Reply:
I'm still updating production as it doesn't require the rhpamcentr image but the rhpamcentrmon one

Reply:
Is this pending?

Reply:
Now it should be ready